deps/llvm.mk

```diff
@@ -515,6 +515,8 @@ $(eval $(call LLVM_PATCH,llvm-D38765-gvn_5.0)) # Remove for 6.0
 $(eval $(call LLVM_PATCH,llvm-D42262-jumpthreading-not-i1))
 $(eval $(call LLVM_PATCH,llvm-PPC-addrspaces)) # PPC
 $(eval $(call LLVM_PATCH,llvm-PR36292-5.0)) # PPC fixes #26249, remove for 6.0
+else ifeq ($(LLVM_VER_SHORT),6.0)
+$(eval $(call LLVM_PATCH,llvm-OProfile-line-num)) # In #27466, to be upstreamed
```

Comment:
You should probably rebase your branch onto current master so that you have the other 6.0 patches as well